Content.Server/HotPotato/HotPotatoSystem.cs

```diff
@@ -52,4 +59,16 @@ private void OnMeleeHit(EntityUid uid, HotPotatoComponent comp, MeleeHitEvent ar
         comp.CanTransfer = false;
         Dirty(comp);
     }
+
+    public override void Update(float frameTime)
+    {
+        var query = EntityQueryEnumerator<ActiveHotPotatoComponent, HotPotatoComponent>();
+        while (query.MoveNext(out var uid, out var _, out var potato))
+        {
+            if (_container.TryGetContainingContainer(uid, out var container))
+            {
+                _damageableSystem.TryChangeDamage(container.Owner, frameTime * potato.HoldingDamage, origin: uid);
+            }
+        }
+    }
```

Comment:
This should be its own component for holding an item doing damage, also it's not a good idea to spam damage events every tick, just tick every second or something.